draft-ietf-dnsop-svcb-https.md

```diff
@@ -659,6 +661,35 @@ each RRSet in the Additional section with the same DNSSEC-related records
 that they would send when providing that RRSet as an Answer (e.g. RRSIG, NSEC,
 NSEC3).
 
+According to Section 5.4.1 of {{!RFC2181}}, "Unauthenticated RRs received
+and cached from ... the additional data section ... should not be cached in
+such a way that they would ever be returned as answers to a received query.
+They may be returned as additional information where appropriate.".
+Recursive resolvers therefore MAY cache records from the Additional section
+for use in populating Additional section responses, and MAY cache them
+for general use if they are authenticated by DNSSEC.
+
+## EDNS Client Subnet (ECS) {#ecs}
+
+The EDNS Client Subnet option (ECS, {{!RFC7871}}) allows recursive
+resolvers to request IP addresses that are suitable for a particular client
+IP range.  SVCB records may contain IP addresses (in ipv*hint SvcParams),
+or direct users to a subnet-specific TargetName, so recursive resolvers
+SHOULD include the same ECS option in SVCB queries as in A/AAAA queries.
+
+According to Section 7.3.1 of {{!RFC7871}}, "Any records from \[the
+Additional section\] MUST NOT be tied to a network".  Accordingly,
+when processing a response whose QTYPE is SVCB-compatible,
```

Comment:
@enygren @MikeBishop I added this line to scope the normative requirement more explicitly.